druid-shell/src/platform/x11/window.rs

```diff
@@ -602,6 +602,45 @@ impl Window {
         }
     }
 
+    pub fn handle_wheel(&self, event: &xcb::ButtonPressEvent) -> Result<(), Error> {
+        let button = event.detail();
+        let mods = key_mods(event.state());
+
+        // We use a delta of 120 per tick to match the behavior of windows.
```

Comment:
Let's call it *Windows* with a capital W to make it clear we're talking about the OS and not the concept, given that all the windows (as in the concept) code is in this same file.